images/kibana/run.sh

```diff
@@ -34,23 +34,26 @@ fi
 chown elastico /opt/kibana/config/kibana.yml
 
 # Adding index to Kibana to set as defaultIndex
-maxretries=60
+SECONDS=0
+maxtime=60
 rc=1
-try=1
-while [ $rc != 0 ]; do
-  echo "Adding index to Kibana (Attempt $try/$maxretries)"
+while [[ $rc -ne 0 ]]; do
+  echo "Attempt to add index to Kibana ($((60 - SECONDS)) sec left)"
```

Comment:
`($((60 - SECONDS))` to `($(($maxtime - SECONDS))`

Reply:
bien vu !